src/pages/HomePage.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import Flex from '../components/common/Flex';
+import Slider from '../components/common/Slider';
+import ImageContainer from '../components/common/Image';
+import Card from '../components/common/Card';
+import Button from '../components/common/Button';
+import { results as productCategoriesResults } from '../utils/product-categories.json';
+import { results as featuredBannersResults } from '../utils/featured-banners.json';
+import { results as featuredProductsResults } from '../utils/featured-products.json';
+
+const Homepage = ({setAtHomePage}) => {
+  const [productsCategories, setProductsCategories] = useState([]);
```

Comment:
You don't need the `useState` and the `useEffect` yet, you can pass them directly to the children's components.

Reply:
Remove `useState` and `useEffect`.